<javaelement>
	<package>
		java.lang
	</package>
	<type>
		<name>
			public class ReflectiveOperationException  extends Exception
		</name>
		<description>
			Common superclass of exceptions thrown by reflective operations in core reflection.
		</description>
		<methods>
			<method>
				<name>
					public ReflectiveOperationException()
				</name>
				<description>
					Constructs a new exception with null as its detail message. The cause is not initialized, and may subsequently be initialized by a call to Throwable.initCause(java.lang.Throwable).
				</description>
				
			</method>
			<method>
				<name>
					public ReflectiveOperationException(String message)
				</name>
				<description>
					Constructs a new exception with the specified detail message. The cause is not initialized, and may subsequently be initialized by a call to Throwable.initCause(java.lang.Throwable).
				</description>
				
			</method>
			<method>
				<name>
					public ReflectiveOperationException(String message,                              Throwable cause)
				</name>
				<description>
					Constructs a new exception with the specified detail message and cause. Note that the detail message associated with cause is not automatically incorporated in this exception's detail message.
				</description>
				
			</method>
			<method>
				<name>
					public ReflectiveOperationException(Throwable cause)
				</name>
				<description>
					Constructs a new exception with the specified cause and a detail message of (cause==null ? null : cause.toString()) (which typically contains the class and detail message of cause).
				</description>
				
			</method>
			
		</methods>
		
	</type>
	
</javaelement>
